tests/profile/benchmark-line-parsing.c

```diff
@@ -643,11 +643,6 @@ void main(void)
     total_active_file_hash = simple_hash("total_active_file");
     total_unevictable_hash = simple_hash("total_unevictable");
 
-    // cache functions
```

Comment:
It's possible that those lines made a difference in small platforms such as raspberry pi. @ktsaou should know why this was added, we should discuss about it before removing them.

Reply:
Sure. However at least the strcmp() and strtoull() calls are made in test1() which is actually called twice with the timing of the second one taken, so they would at least seem redundant in either case.

Reply:
Yeah, this `test1()` was probably coded to run twice at a different date than the cache functions code, so it defeats the purpose in that case.

Reply:
@mfundul just drop a note to @ktsaou to ask him about this, the code does look very strange there.

Reply:
at least it should be commented better

Reply:
I think these are left overs.

The current code now tests `max` iterations. In older versions of it, it used `TSC` CPU commands to test a single iteration. When it was testing just one iteration, it was important the functions to be in the processor caches. Otherwise, loading the function would impact the result significantly.

So, I think we can now remove them.